lib/sass/environment.rb

```diff
@@ -189,15 +189,21 @@ def caller
       @caller ||= env.is_a?(ReadOnlyEnvironment) ? env : ReadOnlyEnvironment.new(env, env.options)
     end
 
-    # The read-only content passed to this environment.
+    # The content passed to this environment but the content's
+    # environment is made readonly if it isn't already.
```

Comment:
I had a lot of trouble parsing this sentence. Maybe "The content passed to this environment. If the content's environment isn't already read-only, it's made read-only."